enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -93,6 +102,7 @@ const std::set<std::string> InactiveGlobals = {
     "stderr",
     "stdout",
     "stdin",
+    "cout",
```

Comment:
This may be too general (e.g. it will match `logicout`). Can you instead move this to inactive starting with the appropriate mangling?

Reply:
done.